server/sttp-stub-server/src/main/scala/sttp/tapir/server/stub/SttpStubServerRequest.scala

```diff
@@ -0,0 +1,16 @@
+package sttp.tapir.server.stub
+
+import java.net.URI
+
+import sttp.client.Request
+import sttp.model.Method
+import sttp.tapir.model.{ConnectionInfo, ServerRequest}
+
+class SttpStubServerRequest[_, S](req: Request[_, S]) extends ServerRequest {
+  override def method: Method = Method(req.method.method)
+  override def protocol: String = "42"
```

Comment:
`"HTTP/1.1"`? ;)

Reply:
Right, this is an easter egg left by the original author ;)